plugins/lsp_python.lua

```diff
@@ -0,0 +1,29 @@
+-- mod-version:3
+
+local lspconfig = require "plugins.lsp.config"
+local common = require "core.common"
+local config = require "core.config"
+local core = require "core"
+
+
+local installed_path_plugin = USERDIR .. PATHSEP .. "plugins" .. PATHSEP .. "lsp_python"
+local server_path = installed_path_plugin .. PATHSEP .. "package" .. PATHSEP .. "langserver.index.js"
+
+
+local ok, node_info = pcall(require, "libraries.nodejs")
+local final_command
+
+if ok then
+  -- NodeJs installed with lpm, runs it.
+  final_command = {node_info.path_bin, server_path, "--stdio"}
+  core.log("[lsp_python]: nodejs library found, using it to launch Pyright.")
```

Comment:
You could use `core.log_quiet` so that it doesn't pollute the status bar. Same thing below.